media/com_joomlaupdate/js/default.js

```diff
@@ -97,34 +150,83 @@ function extractionMethodHandler(target, prefix)
      */
     PreUpdateChecker.setResultView = function (extensionData) {
         var html = '';
-        
-        // Switch the compatibility state
-        switch (extensionData.state) {
-            case PreUpdateChecker.STATE.COMPATIBLE:
-                if (extensionData.compatibleVersion == extensionData.currentVersion) {
-                    // The compatible version matches the current version -> diplay success label.
-                    html = '<span class="label label-success">' + Joomla.JText._('JYES') + '</span>';
-                } else {
-                    // The compatible version does not match the current version => display warning label.
-                    html = '<span class="label label-warning">' + Joomla.JText._('JYES')
-                        + ' (' + extensionData.compatibleVersion + ')</span>';
-                }
-                break;
-            case PreUpdateChecker.STATE.INCOMPATIBLE:
-                // No compatible version found -> display error label
-                html = '<span class="label label-important">' + Joomla.JText._('JNO') + '</span>';
-                break;
-            case PreUpdateChecker.STATE.MISSING_COMPATIBILITY_TAG:
-                // Could not check compatibility state -> display warning
-                html = '<span class="label">' + Joomla.JText._('COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_VERSION_MISSING') + '</span>';
-                break;
-            default:
-                // An error occured -> show unknown error note
-                html = '<span class="label">' + Joomla.JText._('COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSION_WARNING_UNKNOWN') + '</span>';
-        }
+
+        // Process Target Version Extension Compatibility
+        if (extensionData.serverError) {
+			// An error occured -> show unknown error note
```

Comment:
```suggestion
			// An error occurred -> show unknown error note
```
Fix in other instances.